collector/nginx-ingresscontroller/README.md

```diff
@@ -0,0 +1,149 @@
+
+### 1. Create a cluster
+
+First you'll need to create a cluster by a method of your choice. `kind create cluster` works well on Linux and is satisfactory for our purposes on MacOS.
+
+### 2. Install the OTEL collector operator 
+
+#### a. Installation prerequisite 
+
+The OTEL collector operator depends on cert-manager, so we install that first.
+
+```sh
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+```
+
+Since the Collector operator depends on the condition of cert-manager, we'll wait on the prerequisites before we proceed.
+
+```
+kubectl wait deployment -n cert-manager cert-manager --for condition=Available=True --timeout=90s 
+kubectl wait deployment -n cert-manager cert-manager-caininjector --for condition=Available=True --timeout=90s 
+kubectl wait deployment -n cert-manager cert-manager-webhook --for condition=Available=True --timeout=90s 
+```
+
+#### b. Collector installation
+
+Installing the Collector is straightforward. As we usually do when installing Helm charts, we'll start by adding the chart collection to our helm repo and updating.
+
+Now we can either install the OpenTelemetry Operator by applying the manifest like...
+
+```sh
+kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+```
+
+Or we can use the helm charts like ... 
+
+```sh
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+helm repo update
+helm install your-release-name -n your-collector-operator-namespace --create-namespace
+```
+
+3. Install the [NGINX Ingress Controller Operator](https://github.com/nginxinc/nginx-ingress-helm-operator#readme).
+
+The NGINX Ingress Controller Operator is a Helm based operator created with the [Operator Framework](https://sdk.operatorframework.io/). 
+
+Installing the NGINX Ingress Controller Operator requires some manual steps at this time. First we need a copy of the repo. Then we can use the Makefile in that repository root to complete the installation. We do that in this sequence of commands.
+
+```sh
+	git clone https://github.com/nginxinc/nginx-ingress-helm-operator/
+	cd nginx-ingress-helm-operator/
+	git checkout v1.0.0
+	make deploy IMG=nginx/nginx-ingress-operator:1.0.0
+```
+
+This action is in this repo's Makefile by the rule `install-nginx-ingress-operator`.
+
+5. Deploy an NGINX Ingress Controller instance
+
+The file at `ingress/values.yaml` tells the NGINX Ingress Controller Operator how to operate our instance. And `ingress/default-server-secret.yaml` provides a TLS cert for the purpose of the demo. 
+
+```sh
+kubectl apply -f ingress/
+```
+
+For purposes of monitoring via the Prometheus endpoint, we need to tend to two places in our configuration. In the first place we need to customPorts to our service. This needs to map the port where Prometheus metrics are exposed to a port where it will be exposed in the Kubernetes Service.
```

Comment:
```suggestion
For purposes of monitoring via the Prometheus endpoint, we need to tend to two places in our configuration. In the first place we need to add customPorts to our service. This needs to map the port where Prometheus metrics are exposed to a port where it will be exposed in the Kubernetes Service.
```